docs/intro-to-lora.md

```diff
@@ -141,6 +141,31 @@ Now that we have set up the gateways and they can reach the internet, it's time
 
 <span class="tips">**Tip:** LORIOT has a 'Gateway Tap', which you can access from your gateway page. This allows you to see what messages are being picked up by the gateway - very useful when debugging.</span>
 
+### Automatically starting the LORIOT binary when the gateway starts
+
+#### Kerlink IoT station
+
+If you follow the installation steps in the LORIOT documentation, the binary will automatically start whenever the gateway boots. No additional steps required.
```

Comment:
Follow or followed? I assume "followed", since you're not linking to it here so it probably already happened.